season-scraper/leagues/americanHockeyLeague.js

```diff
@@ -0,0 +1,99 @@
+//ahl scraper for npm module
+
+const utils = require('../../utils')
+
+function getSeasons(prospect) {
+  const unFilteredSeasons = [...prospect.careerStats[0].sections[0].data, ...prospect.careerStats[0].sections[1].data].map((season) => season.row)
+  
+  //The raw data has a season called 'total' in both arrays.  Removing it for now
+  const filteredSeasons = unFilteredSeasons.filter((season) => season.season_name != 'Total')
+  return filteredSeasons.map((season) => getFormattedSeason(season))
+
+}
+
+function getInfo(info) {
+  const age = utils.calculation.ageOnDate(info.birthDate)
+
+  return {
+    number: info.jerseyNumber,
```

Comment:
`+info.jerseyNumber` to turn it into integer

Reply:
Yeah I actually didn't even know that was a thing until I saw it in your other code and googled it.  Why did you want to do parseint/float on the other one?

Reply:
Also I probably should be grabbing first/last name there as well.

Reply:
Honestly I don't think it really matters parseInt or + for me, whichever you prefer. I think you need parseFloat for the shootout percentage one because it'll be a decimal

Reply:
> Also I probably should be grabbing first/last name there as well.

Yeah for sure